components/builder-web/app/shared/docker-export-settings/docker-export-settings.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, Input, Output, EventEmitter, OnChanges } from "@angular/core";
+import { FormControl } from "@angular/forms";
+
+@Component({
+  selector: "hab-docker-export-settings",
+  template: require("./docker-export-settings.component.html")
+})
+export class DockerExportSettingsComponent implements OnChanges  {
+  @Input() integrations: any;
+
+  private enabled: boolean = false;
+  private name: string;
+  private repoName: string = "";
+  private customTag: string;
+  private latestTag: boolean = true;
+  private versionTag: boolean = true;
+  private releaseTag: boolean = true;
+
+  get configured() {
+    return this.integrations.size > 0;
+  }
+
+  get settings(): any {
+    return {
+      valid: this.valid,
+      name: this.name,
+      enabled: this.enabled,
```

Comment:
Does having the enabled flag toggle delete the settings?  We might want to pass in enabled into the settings body as well, so that the user does not have to re-enter the settings.  Ignore if this is not the way things are working, and the worker will get to see the enabled setting.

Reply:
@chefsalim that's just an instruction for the UI to know whether the user has toggled the slider to enable the integration, yeah. It doesn't get sent in the request body. I was planning to use it to toggle a delete once we have an support to support that.

Reply:
So just to confirm, with the current code, once a Docker integration is enabled, it can not effectively be disabled since we don't pass that setting into the integration itself (and currently don't delete it)

Reply:
That's correct, although we do expose UI to delete the *project* -- we just aren't able to delete the project integration settings yet.